optin/puller/workflows/get_account_states.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import asyncio
+import sqlite3
+from pathlib import Path
+
+from aiolimiter import AsyncLimiter
+from symbolchain.nem.Network import Address
+
+from puller.client.NemClient import NemClient
+from puller.db.InProgressOptinDatabase import InProgressOptinDatabase
+from puller.db.BalancesDatabase import BalancesDatabase
+
+SNAPSHOT_HEIGHT = 3105500
+
+
+def get_addresses(filename):
+	with sqlite3.connect(f'file:{filename}?mode=ro', uri=True) as connection:
+		in_progress = InProgressOptinDatabase(connection)
```

Comment:
inprogress_database or in_progress_database ?

Reply:
hmm, thought `in_progress`, but not sure now :D

Reply:
at least add `_database` postfix